libdevcore/CommonData.h

```diff
@@ -188,12 +188,6 @@ inline bytes toCompactBigEndian(uint8_t _val, unsigned _min = 0)
 	return (_min || _val) ? bytes{ _val } : bytes{};
 }
 
-/// Workarounds shift left bug in boost <1.65.1.
```

Comment:
Why move this and not keep it here?

Reply:
It is not used anywhere else and all the other workarounds are also in the rules.

Reply:
How? There were rules in RuleList which used it?

Reply:
Sure, I changed them to use the implementation inside RuleList.